src/Microsoft.VisualStudio.LanguageServices.Razor/DefaultTextViewRazorDocumentTrackerService.cs

```diff
@@ -165,5 +166,12 @@ public virtual bool IsSupportedProject(IVsHierarchy project)
             
             return project.IsCapabilityMatch("DotNetCoreWeb");
         }
+
+        public static IEnumerable<ITextView> GetTextViews(ITextBuffer textBuffer)
```

Comment:
This is temporary until [this](https://github.com/aspnet/Razor/issues/1633) is completed. Basically the parser needs to know about the views associated with a TextBuffer and we need to capture that information.